src/contracts/Chef.sol

```diff
@@ -41,6 +45,24 @@ abstract contract Chef is Ownable {
         return MetricToken(metricTokenAddress);
     }
 
+    function setLifetimeShareValue() public virtual {
+        uint256 accumulated = getAccumulated();
+        uint256 accumulatedWithMetricPrecision = getAcculatedWithmetricPrecision(accumulated);
+        _lifetimeShareValue = _lifetimeShareValue.add(accumulatedMetricDividedByShares(accumulatedWithMetricPrecision));
+    }
+
+    function setTotalAllocShares(uint256 shares) internal virtual {
```

Comment:
can we rename this to `addTotalAllocShares`?  reading the subclass I had to come back and check the implementation 🙃

Reply:
done